MAUI/DataForm/DataForm-settings.md

```diff
@@ -310,7 +310,7 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
 {% endhighlight %}
 {% endtabs %}
 
-![PlaceHolderTextColor](images/editor-settings/placeholder-text-color.png)
+![Placeholder text color for .NET MAUI DataForm.](images/editor-settings/placeholder-text-color.png)
```

Comment:
Editor placeholder text color for .NET MAUI DataForm.

Reply:
changed